### PR TITLE
Supabase case study.md

### PR DESCRIPTION
**Pull request by martemoslet**
## Release Notes

- Documentation: Adds a case study of Supabase, including an introduction, brief history, main features, strengths, weaknesses, comparison with Firebase, and conclusion. Includes references to various sources.

> "Supabase shines bright,
> With PostgreSQL and more in sight.
> A case study now in view,
> Thanks to this PR's crew."

### CODE REVIEW

File: supabase-case-study.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+ ## Introduction
+
+ Supabase is a backend-as-a-Service (BaaS) platform that offers PostgreSQL database, authentication, storage and edge functions. It was created in 2019 as an open-source alternative to Firebase. 
+
+ ## Brief history
+
+- 2019: Supabase is created
+- July 2020: Announced Auth
+- November 2020: Supabase Beta
```
Comment (Fermain) [inline supabase-case-study.md:15]: In December 2020 Supabase received $6m in funding.

There is a great article on the history here: https://www.topofthelyne.co/p/how-supabase-is-taking-on-the-goliath

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a good quality submission with a few required changes and some formatting to clean up. Consider using the linting plugin along with chatGPT to clean up your markdown formatting and catch any remaining grammar issues.

File: supabase-case-study.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+title: Supabase Case Study
+author: Marte Moslet <martemoslet>
+tags: supabase, case study, platform, BaaS
+---
+
+ ## Introduction
```
[the remark this answers is not in the document]
Comment (martemoslet) [reply supabase-case-study.md]: Thank you. I have now removed the spacing and formatted with markdownlint. And also checked grammar and spelling with chatGPT.

File: supabase-case-study.md (hunk 1 above)
Comment (martemoslet) [reply supabase-case-study.md:15]: Thanks, I have now added that fact

**Review by Fermain — APPROVED** [review]

**Pull request merged**